README.md

```diff
@@ -1,11 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# COVID-19 UNIQUE DATA
+> Unique data is a react project that provides COVID-19 data for each country. It is a SPA website provides users with an immediate update on COVID-19 data. It showcases my work at the end of the react milestone.
```

Comment:
- [ ] please ensure to add an acknowledgments section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).